Code/GraphMol/testChirality.cpp

```diff
@@ -2784,6 +2784,113 @@ void testStereoGroupUpdating() {
   TEST_ASSERT(m->getStereoGroups().size() == 0u);
 }
 
+class TestAssignChiralTypesFromMolParity {
```

Comment:
Same comment here about test complexity. Is there a reason you can't just add a simple test for what you know the result should be?

Reply:
Ditto.